### DISCUSSION TITLE
MorphTo Select in Form

### DISCUSSION THREAD

**Discussion by invaders-xx** [filamentphp/filament#2117] [Help]
Is there a way to create a select which belongs to a MorphTo relation, or do I need to a create a select for each type of morph? (which can be a mess). Many thanks for your help.

**Comment by ryangjchandler** [comment]
If you have a fixed list of possible classes that can be morphed to, it's possible with a custom searchable select.

**Comment by invaders-xx** [comment]
I finally did that with 2 dependant select : first with type and second with ids, updated depending on the first. it works
